Character/Classes/Sorcerer/Sorcerer.xml

```diff
@@ -189,7 +191,7 @@
 		</autolevel>
 		<autolevel level="8" scoreImprovement="YES">
 		</autolevel>
-		<autolevel level="10">
+		<autolevel level="10" IL="IL">
```

Comment:
Same as 3, IL doesn't change anything here

Reply:
Per the other comment on 17, this is fine, but should be moved down as to not also eliminate the Metamagic Option feature